client/landing/stepper/declarative-flow/internals/hooks/use-launchpad-decider/index.tsx

```diff
@@ -21,8 +20,7 @@ interface SiteProps {
 }
 
 export const useLaunchpadDecider = ( { exitFlow, navigate }: Props ) => {
-	const [ isLoadingExperiment, experimentAssignment ] = useExperiment( LAUNCHPAD_EXPERIMENT_NAME );
-	const showCustomerHome = shouldShowCustomerHome( isLoadingExperiment, experimentAssignment );
+	const showCustomerHome = true;
```

Comment:
Could you help me understand why this is set to `true`?

Reply:
In local development, I tested and switched the `showCustomHome` value. It was the end of the day, and I didn't realize that I pushed the wrong value. :(

Sorry about that. Now, should be fine